labellab_mobile/lib/widgets/issue_list_tile.dart

```diff
@@ -1,208 +1,200 @@
+                                  _getBackgroundColor(
+                                      IssueMapper.priorityToString(
+                                          issue.issuePriority))),
+                              SizedBox(
+                                width: 8,
+                              ),
+                            ],
                           ),
-                        ),
+                        )
                       ],
                     ),
                   ),
-                  SizedBox(
-                    width: size.width / 4,
-                  ),
-                  Column(
-                    mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                    children: [
-                      // Text("Issue #" +issue.id!.toString()),
-                      Text(
-                        "Priority: " +
-                            IssueMapper.priorityToString(issue.issuePriority),
-                        style: TextStyle(
-                          fontSize: 12,
-                          color: isCustomized
-                              ? _getPriorityTextColor(
-                                  IssueMapper.priorityToString(
-                                      issue.issuePriority))
-                              : Color(0xff01A8A0),
-                        ),
-                      ),
-                      Text(
-                        "Category: " +
-                            IssueMapper.categoryToString(issue.issueCategory),
-                        style: TextStyle(
-                          fontSize: 12,
-                          color: isCustomized
-                              ? _getTextOrBorderColor(
-                                  IssueMapper.categoryToString(
-                                      issue.issueCategory))
-                              : Color(0xff01A8A0),
-                        ),
-                      )
-                    ],
-                  )
-                ],
+                ),
+                flex: 8,
               ),
             ],
           ),
-          Padding(
-            padding: const EdgeInsets.only(left: 120),
-            child: Text(issue.issueTitle!,
-                style: TextStyle(
-                  fontSize: 16,
-                  fontWeight: FontWeight.bold,
-                )),
-          ),
-          Divider(
-            height: 5,
-            color: Colors.black,
-          ),
-          Row(
-            mainAxisAlignment: MainAxisAlignment.spaceBetween,
-            children: [
-              Text("Posted by: " + issue.created_by.toString(),style: TextStyle(
-                          fontSize: 12,)),
-              Text("Comments " + 0.toString(),style: TextStyle(
-                          fontSize: 12,)),
-            ],
-          )
-        ],
+        ),
+        elevation: 8,
+        margin: EdgeInsets.all(10),
       ),
     );
   }
 
   Color _getBackgroundColor(String category) {
```

Comment:
Variable name should be `priority` or something similar. `category` does not make sense here.

Reply:
If it's a priority specific function, name the variable `priority` otherwise something generic like `type`.